WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -147,7 +137,32 @@ private extension ShippingLabelAddressFormViewController {
     }
 
     func updateTopBannerView() {
-        topBannerView.isHidden = !viewModel.shouldShowTopBannerView
+        if !viewModel.shouldShowTopBannerView {
+            hideTopBannerView()
+        }
+        else {
+            displayTopBannerView()
+        }
+    }
+
+    func displayTopBannerView() {
+        // Configure header container view
+        let headerContainer = UIView(frame: CGRect(x: 0, y: 0, width: Int(tableView.frame.width), height: 0))
+        headerContainer.addSubview(topStackView)
+        headerContainer.pinSubviewToSafeArea(topStackView)
+        topStackView.addArrangedSubview(topBannerView)
+
+        tableView.tableHeaderView = headerContainer
+        tableView.updateHeaderHeight()
+    }
+
+    func hideTopBannerView() {
+        guard tableView.tableHeaderView != nil else {
+            return
+        }
+
+        topBannerView.removeFromSuperview()
+        tableView.tableHeaderView = nil
         tableView.updateHeaderHeight()
```

Comment:
nit: I don't think we need `updateHeaderHeight()` here, since we've already set `tableHeaderView` to `nil` in the line above.